.grit/patterns/python/warning-for-tempfile-without-flush.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Warning for tempfile without flush
+---
+
+"Be cautious when using `$F.name` without preceding it with `.flush()` or `.close()`, as it may result in an error. This is because the file referenced by `$F.name` might not exist at the time of use. To prevent issues, ensure that you either call `.flush()` to write any buffered data to the file or close the file with .close() before referencing `$F.name`.
```

Comment:
Don't just copy and paste quotes from the docs. At the very least the leading quote mark isn't helpful and confuses this.